cdk/typescript/lib/the-lambda-power-tuner-stack.ts

```diff
@@ -0,0 +1,41 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import * as sam from 'aws-cdk-lib/aws-sam';
+
+export class TheLambdaPowerTunerStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    let powerValues = '128,256,512,1024,1536,3008';
+    let lambdaResource = "*";
+
+    // A lambda function to use to test the powertuner
+    let exampleLambda = new lambda.Function(this, 'lambdaHandler', {
```

Comment:
Personally, I think this is also optional :) you don't necessarily have to define the Lambda function you're power-tuning in the same stack. Actually, most people just deploy Lambda Power Tuning once and then use it with as many functions as needed. 

I understand this is a sample pattern and customers will be able to customize it as needed, but I'd try to make it very clear that they don't have to deploy their functions in the same stack.